### ISSUE TITLE
Does not load current language

### ISSUE THREAD

**Issue by rraczo** [Automattic/liveblog#345]
The texts are loaded from the js files, so it does not load the different languages.

EntryContainer.js
```
<button className="liveblog-btn liveblog-btn-small liveblog-btn-delete" onClick={this.delete}>
	Delete
</button>
```

I think they should change it for something like this:

```
<button className = "liveblog-btn liveblog-btn-small liveblog-btn-delete" onClick = {this.delete}>
     <? php _e ('Delete', 'liveblog'); ?>
</ button>
```

**Comment by jmslbam** [comment]
Yupp same here.

Strings like buttons and other elements aren't translated anymore.
There is also a Support ticket for this: https://wordpress.org/support/topic/translation-855/

-------
Clean VVV
Latest Theme: Twenty SeventeenVersion: 1.7
No other plugins
Liveblog: Version 1.8.2
WP: 4.9.7
Language settings: Dutch / Nederlands

**Comment by cain** [comment]
I'm guessing the easiest fix is to use a javascript plugin and pass down the language in WordPress to redux. eg: https://github.com/i18next/react-i18next

**Comment by earcos** [comment]
This is crazy. Translation worked fine on previous versions.

**Comment by cain** [comment]
@earcos Which version?

**Comment by earcos** [comment]
Any version prior to 1.7.

**Comment by cain** [comment]
@earcos I'm not fully across live blog prior to 1.8, but it looks like they moved away from templating in PHP to a react frontend. If you need to support languages in the current version you'll need to wait, or you can work on it yourself. I can help if you want to get started or have any questions.

**Comment by earcos** [comment]
@cain that means the documentation is outdated? 😢

**Comment by cain** [comment]
Yes, some of it is outdated. The amp template might possibly support languages but you'll have to double check.

**Comment by jamunfer** [comment]
I run CNNE.com and we are very interested in the Spanish. @earcos
